src/libraries/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -476,10 +476,10 @@ internal void LoadIntoBuffer(long maxBufferSize, CancellationToken cancellationT
         public Task LoadIntoBufferAsync(long maxBufferSize) =>
             LoadIntoBufferAsync(maxBufferSize, CancellationToken.None);
 
-        internal Task LoadIntoBufferAsync(CancellationToken cancellationToken) =>
+        public Task LoadIntoBufferAsync(CancellationToken cancellationToken) =>
```

Comment:
Please add XML comments to these. I realize the other existing APIs don't have them, but the comments will be used to seed the docs.

Reply:
Documentation added.